### PR TITLE
Add Empower (MPWR) on Base

### PR DESCRIPTION
**Pull request by pauljiwoongchung**
We're looking to add MPWR to Base.

Etherscan: https://etherscan.io/token/0x6731827Cb6879a2091ce3ab3423f7bf20539b579
Etherscan Goerli: https://goerli.etherscan.io/address/0x1E1dfcf72dceF548e6AE581863a564FFe429dAA8 
Basescan: https://basescan.org/address/0x85267fBC6e1e1563bB9247336F05D9220eA99f52
Basescan Goerli: https://goerli.basescan.org/address/0x1901Bf2952F438Fa6251ac100375769f9Afd907d

@roberto-bayardo

### CODE REVIEW

**Comment by pauljiwoongchung** [comment]
Hi @roberto-bayardo 

Could you please review the request? I'm trying to add 'Base' label into this PR to get a review from you, but I couldn't add/modify the label to 'Base'

If you need anything on my end, please let me know about it.

**Review by roberto-bayardo — DISMISSED** [review]
Good to go from Base side.

**Comment by pauljiwoongchung** [comment]
Dear @roberto-bayardo 

Thank you for approving the review. I noticed an error message popping up after you approved the review, which said "Rule: automatic update prs (update) — Pull request can't be updated with latest base branch changes". Because of this, I updated the my forked repository to the latest content to fix it. Unfortunately, it seems like this action caused Mergify to dismiss your review.

I'm really sorry for the inconvenience, but could you please approve the review once again?

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**
